_posts/2021-02-19-understanding-NOPE.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: post
+title: Understanding NOPE
+categories: [NOPE, market makers, delta hedging]
+excerpt: An attempt to give you a working understanding of what NOPE is and the theory behind it. Requires some options knowledge.
+---
+
+This post aims to provide a working understanding of what NOPE is and the theory behind it. I will cover:
+
+- Who is on the other side of your option trades
+- How they avoid being stuck holding the opposite position to yours
+- What this does to the market
+- What NOPE is measuring and how it does it
+
+## Prerequisites
+
+This post is aimed at someone with a beginner-to-intermediate understanding of options trading. Specifically, I assume you already know:
+
+- What options are
+- What the order book is
+- A basic understanding of the options greeks
+- That the Black-Scholes formula exists and that it's used to price options using the greeks. (But don't need to know it or be able to derive it!)
```

Comment:
Perhaps mention here that there are other pricing models as well. BSM is...less than perfect

"That the BSM (and other newer better) formulas exist to price options using the greeks."